cms/tests/test_plugins.py

```diff
@@ -1102,7 +1102,7 @@ def test_copy_plugin_with_m2m(self):
             language=self.FIRST_LANG,
         )
 
-        endpoint = self.get_admin_url(Page, 'edit_plugin', plugin.pk)
+        endpoint = self.get_admin_url(Placeholder, 'edit_plugin', plugin.pk)
```

Comment:
replace any usage of get_admin_url for plugin endpoints with the respective get_<action>_plugin_uri helper method